src/components/Link/Link.module.scss

```diff
@@ -0,0 +1,37 @@
+@use '../../styles/colors.module';
+@use '../../styles/mixins.module';
+
+.link {
+    display: inline-block;
+    padding: 0.5rem 1rem;
+    border-radius: 64px;
+    color: colors.getColor(warning-400);
+
+    &:hover {
+        //background-color: colors.getColor(warning-50);
+        background-color: rgba($color: #FFFBEB, $alpha: 0.12);
```

Comment:
please use the alpha parameter of getColor instead of hard coding the color:
eg.
```
colors.getColor(warning-50, null, 0.12)
```